warrior/WarriorCore/Classes/katana_interface_class.py

```diff
@@ -0,0 +1,38 @@
+from xml.dom import minidom
+import os
+import json
+try:
+    import requests
+    has_requests = True
+except ImportError:
+    has_requests = False
```

Comment:
Need an error message showing that this functionality isn't activated when requests isn't installed